
bool detect_obstacles
bool slow_down
float64 angular_velocity
float64 linear_velocity
